lib/include-all.js

```diff
@@ -52,7 +51,7 @@ IncludeAll.prototype.importPods = function(patches, srcDir) {
   for (var extension in this.importedPods) {
     var output = '';
     for (var file in this.importedPods[extension]) {
-      output += this.importedPods[extension][file];// + ';' + os.EOL;
+      output += this.importedPods[extension][file] + ';' + os.EOL;
```

Comment:
Propably would need to conditionally, or via some kind of dictionary lookup, include or not include the semi-colon.